Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7261,7 +9249,9 @@ def main():
 
         if demisto.command() == 'test-module':
             panorama_test()
-
+            # Test the topology is working as well
+            topology = get_topology()
+            test_topology_connectivity(topology)
```

Comment:
This still needs to be fixed. I still think we should have a 'test_module' function where both panorama_test and test_topology_connectivity are called. could there be a customer which uses panorama commands but won't use the topology? if yes lets discuss

Reply:
Yes, there would definitely be customers that don't need the topology commands (and vice versa).